ChessBoard.Api/ChessBoard.Api.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <UserSecretsId>35dca561-0268-480c-bd20-249f590e7294</UserSecretsId>
+    <DockerDefaultTargetOS>Windows</DockerDefaultTargetOS>
```

Comment:
Just a side note - most of containers used in docker env are Linux containers, let's stick to that ;) No need to adjust at this time.

Reply:
Understood